examples/interactionsDemo.html

```diff
@@ -0,0 +1,532 @@
+                    clientApp.conversations.showInteractionDetails(conversationId);
+                });
+
+                // Authenticate with PureCloud
+                let authenticatingEl = document.querySelector('.authenticating');
+                let userDetailsEl = document.querySelector('.userDetails');
+
+                setHidden(authenticatingEl, false);
+
+                let authenticated = false;
+                let userDataAcquired = false;
+                let canViewInteraction = false;
+                let canViewWrapUpCodes = false;
+
+                client.loginImplicitGrant(clientId, redirectUrl, { state: ('pcEnvironment=' + pcEnvironment) })
+                    .then(function () {
+                        authenticated = true;
+                        return new platformClient.UsersApi().getUsersMe({ 'expand': ['authorization'] });
+                    })
+                    .then(function (profileData) {
+                        userDataAcquired = true;
+
+                        // Update user data markup
+                        $('#name').append(' ' + profileData.name);
+
+                        // Check if the current user will be able to even view the interaction before making the request
+                        let permissions = profileData.authorization.permissions;
+                        if (checkPermission(permissions, 'analytics:conversationDetail:view') &&
+                            (checkPermission(permissions, 'conversation:communication:view') ||
+                             checkPermission(permissions, 'quality:evaluation:add') ||
+                             checkPermission(permissions, 'quality:calibration:view') ||
+                             checkPermission(permissions, 'quality:evaluation:editScore'))) {
+                                canViewInteraction = true;
+                                $('.interactionDetailAccess').append(' Yes');
+                        } else {
+                                $('.interactionDetailAccess').append(' No');
+                        }
+
+                        if (checkPermission(permissions, 'routing:wrapupCode:view')) {
+                            canViewWrapUpCodes = true;
+                        }
+
+                        setHidden(authenticatingEl, true);
+                        setHidden(userDetailsEl, false);
+
+                        $('#reloadButton').on('click', function () {
+                            $('.failure').text('');
+                            $(this).attr('disabled', '');
+                            setHidden(reloadingEl, false);
+                            $('#tableBody tr').remove();
+
+                            // 30 days in the past
+                            let today = new Date();
+                            let today30 = (new Date(new Date().setDate(today.getDate() - 30))).toISOString();
+
+                            return (function() {
```

Comment:
Any reason to return here?  I don't think jquery will care about anything except truthy/falsy here and would only prevent the default.  Not sure if this was an attempt to block clicks based on the resolution of the promise for instance.

Since the refresh button is only re-enabled on the promise resolution/rejection, we shouldn't need to block otherwise.

Could debounce the click handler, but probably overkill for a demo app.

Reply:
TBH, I do not recall why I returned here but as you said it is not needed. I can pull it out.

Reply:
👍